### DISCUSSION TITLE
Json code issue: - I keep getting a DateAdd error message on a sharepoint list when I use the following json code; could you please assist me? Raw

### DISCUSSION THREAD

**Discussion by Lee996677** [community/community#49021] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

{
  "$schema": "https://developer.microsoft.com/json-schemas/sp/v2/column-formatting.schema.json",
  "debugMode": true,
  "elmType": "div",
  "style": {
    "color": {
      "operator": "?",
      "operands": [
        {
          "operator": "<=",
          "operands": [
            "@currentField",
            {
              "operator": "DateAdd",
              "operands": [
                {
                  "operator": "Now"
                },
                {
                  "operator": "Days",
                  "operands": [
                    7
                  ]
                }
              ]
            }
          ]
        },
        "red",
        {
          "operator": "?",
          "operands": [
            {
              "operator": "<=",
              "operands": [
                "@currentField",
                {
                  "operator": "DateAdd",
                  "operands": [
                    {
                      "operator": "Now"
                    },
                    {
                      "operator": "Days",
                      "operands": [
                        15
                      ]
                    }
                  ]
                }
              ]
            },
            "orange",
            ""
          ]
        }
      ]
    }
  },
  "txtContent": "@currentField"
}

**Comment by Lee996677** [comment]
I keep getting a DateAdd error message on sharepoint list with the following json code, can anyone help me please?

**Failure: 'Now' is not a valid operator. It must be one of toString0 cos floor ceiling sin abs Date Number toLowerCase toUpperCase length loopIndex to DateString() toLocaleString 0
toLocaleDateString0 toLocaleTimeString0
getDate getMonth getYear = = != > = <=

< + getUserlmage - * / % pow || 88 addDays addMinutes indexOf
lastIndexOf startsWith endsWith join split appendTo removeFrom in : ? substring replace replaceAll padStart getThumbnaillmage padEnd in the expression {"operator". "Now"}.**

**Comment by ettaboyle** [comment]
Hi @Lee996677,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
